content/post/057-ipfs-desktop-0-9.md

```diff
@@ -0,0 +1,65 @@
+---
+date: 2019-09-13
+url: 057-ipfs-desktop-0-9
+title: IPFS Desktop 0.9 released
+author: Henrique Dias
+---
+
+IPFS Desktop has come a long way to being what it is nowadays. We are here to cellebrate and announce the [release of 0.9.0](https://github.com/ipfs-shipyard/ipfs-desktop/releases) and, with that, tell a bit about the story of this application and, of course, the new features.
+
+## Once upon a time...
+
+It was back in May 2015 when Juan Benet pushed a first so-called "Initial Commit" to start a project at the time named "ipfs-electron". The purpose of the app at the time was quite simple and that hasn't changed much with time: providing a long running daemon, alongside with a GUI for managing it.
+
+> TODO: add timeline
+
+Regarding the application's name, it went from "ipfs-electron" to "IPFS Native Application", then to "IPFS Station", it being a much more fancier name. However, we changed it again to "IPFS Desktop" and that's where we are right now!
+
+> TODO: screenshots of older versions
+
+## IPFS Desktop 0.9.0
+
+The newest version of IPFS Desktop, 0.9.2 - yes, we already released two patch versions! - brings more stability and some new experimental features. Although it is not a completely revamped application, such as what happened with 0.7.0, I believe this version deserves an overview of the current (new) features and plans for the future.
+
+### Browse the wild...
+
+With the recent update to the Web UI, we are now capable of navigating throug the whole IPFS system. Previously, you could only access your 'Files' - Mutable File System for those who enjoy technical terms. Now, you can navigate not only to any `/ipfs/` or `/ipns/` path, but also to your pins!
```

Comment:
```suggestion
With the recent update to the Web UI, we are now capable of navigating through the whole IPFS system. Previously, you could only access your 'Files' (the contents of the [Mutable File System](https://docs.ipfs.io/guides/concepts/mfs/) for those who enjoy technical terms). Now, you can navigate not only to any `/ipfs/` or `/ipns/` path, but also any content [pinned](https://docs.ipfs.io/guides/concepts/pinning/) to your IPFS node!
```
I'd suggest adding links to the terminology you use so that beginners can better understand what we mean by these commands. A lot of nontechnical users read our blog - and this is a great onboarding opportunity for them!